src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,43 @@
+.catalog {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: repeat(autofill, 200px);
```

Comment:
this rule is redundant because you have a query @media (max-width: 488px) where you set a rule from 0px to 488px
```suggestion

```